src/main/java/io/kestra/plugin/notion/CreatePage.java

```diff
@@ -0,0 +1,277 @@
+        title = "Page title",
+        description = "The title of the new page"
+    )
+    @NotNull
+    private Property<String> title;
+
+    @Schema(
+        title = "Page content",
+        description = "The content of the page in markdown format"
+    )
+    private Property<String> content;
+
+    @Schema(
+        title = "Parent page ID",
+        description = "Optional parent page ID for creating a child page. If not specified, the page will be created at the workspace root."
+    )
+    private Property<String> parentPageId;
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        Logger logger = runContext.logger();
+
+        // Render and validate inputs
+        String renderedTitle = runContext.render(this.title).as(String.class).orElseThrow(
+            () -> new IllegalArgumentException("title is required")
+        );
+
+        String renderedContent = runContext.render(this.content).as(String.class).orElse("");
+
+        String renderedParentPageId = null;
+        if (this.parentPageId != null) {
+            renderedParentPageId = runContext.render(this.parentPageId).as(String.class).orElse(null);
+            if (renderedParentPageId != null && !renderedParentPageId.isEmpty()) {
+                // Validate parent page ID format
+                if (!renderedParentPageId.matches("^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$") && 
+                    !renderedParentPageId.matches("^[0-9a-f]{32}$")) {
+                    throw new IllegalArgumentException("parentPageId must be a valid Notion page ID (UUID format)");
+                }
+            }
+        }
+
+        // Build the request body
+        Map<String, Object> requestBody = buildCreatePageRequest(renderedTitle, renderedContent, renderedParentPageId);
+
+        // Make the API call
+        String url = buildCreatePageURL();
+        HttpRequest.HttpRequestBuilder requestBuilder = buildPostRequest(runContext, url, requestBody);
+
+        logger.info("Creating Notion page with title: {}", renderedTitle);
+        logger.debug("Request body: {}", mapper.writeValueAsString(requestBody));
+
+        NotionResponse response = makeCall(runContext, requestBuilder, NotionResponse.class);
+
+        // Store detailed response
+        URI fileURI = store(runContext, List.of(Map.of(
+            "response", response,
+            "title", renderedTitle,
+            "content", renderedContent,
+            "parentPageId", renderedParentPageId
+        )));
+
+        return Output.builder()
+            .pageId(response.getId())
+            .url(response.getUrl())
+            .title(renderedTitle)
+            .content(renderedContent)
+            .createdTime(response.getCreatedTime())
+            .lastEditedTime(response.getLastEditedTime())
+            .archived(response.getArchived())
+            .properties(response.getProperties())
+            .uri(fileURI)
+            .success(true)
```

Comment:
This param is always true so useless?

Reply:
fixed by https://github.com/kestra-io/plugin-notion/pull/2/commits/14bee7489735d52ec423ef1142cb031aecbcd506